src/core/pages/InitializePage.js

```diff
@@ -0,0 +1,273 @@
+                                    <p>Watch out for high salt levels in ready-made foods. Most of the salt we eat is already in the foods we buy. Check the food labels – a food is high in salt if it has more than 1.5g salt (or 0.6g sodium) per 100g.</p>
+                                    <p>Adults should eat less than 6g of salt a day in total – that's about 1 teaspoon.</p>
+                                </div>
+                            </div>
+
+                            <div className={classes.slide}>
+                                <div className={classes.slideNumber}>
+                                    <h3>Eat Fish</h3>
+                                    <p>Eat fish at least twice a week, including a portion of oily fish. Fish such as pilchards, sardines and salmon are a source of omega-3 fats, which may help protect against heart disease.</p>
+                                    <p>Pregnant or breastfeeding women should not have more than 2 portions of oily fish a week.</p>
+                                </div>
+                            </div>
+
+                            <div className={classes.slide}>
+                                <div className={classes.slideNumber}>
+                                    <h3>Drink less alcohol</h3>
+                                    <p>Do not forget that alcohol contains calories. Regularly drinking more than the NHS recommends can have a noticeable impact on your waistline.</p>
+                                    <p>Try to keep to the recommended daily alcohol limits to reduce the risk of serious problems with your health, including risks to your heart health.</p>
+                                </div>
+                            </div>
+
+                            <div className={classes.slide}>
+                                <div className={classes.slideNumber}>
+                                    <h3>Read the food label</h3>
+                                    <p>When shopping, it's a good idea to look at the label on food and drink packaging to see how many calories and how much fat, salt and sugar the product contains.</p>
+                                    <p>Understanding what's in food and how it fits in with the rest of your diet will help you make healthier choices.</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    }
+                    <div className={classes.rings}>
+                        <div></div>
+                        <div></div>
+                    </div>
+                </div>
+                <div className={classes.progress}>
+                    <div className={classes.progressBar} role="progressbar" style={{width: 0 + '%'}}></div>
```

Comment:
Why use `style` here instead of a class?

Reply:
Removed